tests/CMakeLists.txt

```diff
@@ -47,6 +49,9 @@ if (UNIX OR ADD_WINDOWS_ENCLAVE_TESTS OR USE_CLANGW)
         add_subdirectory(enclaveparam)
         add_subdirectory(file)
         add_subdirectory(getenclave)
+        add_subdirectory(getenclave_extend)
+        add_subdirectory(getenclave_nondebugenc)
+        add_subdirectory(atexit)
```

Comment:
This one too

Reply:
removed atexit and getenclave_nondebugenc. And getenclave_nondebugenc was merged to getenclave_extend.